test/intl402/Locale/constructor-options-numeric-valid.js

```diff
@@ -46,21 +46,23 @@ const validNumericOptions = [
   [{ valueOf() { return false; } }, true],
 ];
 for (const [numeric, expected] of validNumericOptions) {
-  let options = { numeric };
   let expect = `en-u-kn-${expected}`;
 
   assert.sameValue(
-    new Intl.Locale('en', options).toString(),
-    expect
+    new Intl.Locale('en', {numeric}).toString(),
+    expect,
+    `new Intl.Locale("en", {numeric: ${numeric}}).toString() returns "${expected}"`
   );
 
   assert.sameValue(
-    new Intl.Locale('en-u-kn-true', options).toString(),
-    expect
+    new Intl.Locale('en-u-kn-true', {numeric}).toString(),
+    expect,
+    `new Intl.Locale("en-u-kn-true", {numeric: ${numeric}}).toString() returns "${expected}"`
   );
 
   assert.sameValue(
-    new Intl.Locale('en-u-kf-lower', options).numeric,
-    expected
+    new Intl.Locale('en-u-kf-lower', {numeric}).numeric,
+    String(expected),
```

Comment:
@rwaldron: You reviewed and landed my change in https://github.com/tc39/test262/pull/1688, and then reverted it within a day without even giving me a chance to review? Can you please explain to me how this is acceptable behavior?